File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
       int n=nums.length;
       int actual_sum=0;
       for(int i=0;i<n;i++){
actual_sum+=nums[i];
       } 
      int sum=n*(n+1)/2;
       return sum-actual_sum;
    }
}

File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int i = 1;
        if (nums[0] != 0)
            return 0;
        for (i; i < nums.size(); i++) {
            if (nums[i - 1] + 1 != nums[i])
                return nums[i - 1] + 1;
        }
        return nums[i - 1] + 1;
    }
};